canban-react/src/components/Column.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import "../style.css";
+import Card from './Card.js';
+import PropTypes from 'prop-types';
+
+export default class Column extends React.Component{
+	addCards(){
+		let arrBuff=[];
+		let arrCardData = this.props.cardsData;
+		for(let i=0; i<arrCardData.length; i++){
+			arrBuff.push(<Card key={arrCardData[i].id} 
+							   id={arrCardData[i].id} 
```

Comment:
you don't need to pass each parameter as a separated prop, you can pass only 1 prop for that ("card" or something like this)